bitcoin/src/blockdata/transaction.rs

```diff
@@ -35,6 +35,11 @@ use crate::sighash::{EcdsaSighashType, TapSighashType};
 use crate::string::FromHexStr;
 use crate::{io, Amount, VarInt};
 
+// The marker MUST be a 1-byte zero value: 0x00. (BIP-141)
+const MARKER: u8 = 0x00;
+// The flag MUST be a 1-byte non-zero value. Currently, 0x01 MUST be used. (BIP-141)
+const FLAG: u8 = 0x01;
```

Comment:
in 111b344ce10f08b997a56b90a5536c0ae51b855c:

These constants should have a `SEGWIT_` prefix. (I know they're not exposed, and that they have comments that cite BIP 141, but it's still good to clarify that they're only applicable to segwit transactions.)

Reply:
Yes, definitely. Cheers